arch/arm64/Kconfig

```diff
@@ -0,0 +1,39 @@
+# ARM64 architecture configuration options
+
+# Copyright (c) 2014-2015 Wind River Systems, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+menu "ARM Options"
+	depends on ARM64
+
+config ARCH
+	default "arm64"
+
+config ARM64
+	bool
+	select 64BIT
+
+config CPU_CORTEX
+	bool
+	help
+	  This option signifies the use of a CPU of the Cortex family.
+
+config ARM_CUSTOM_INTERRUPT_CONTROLLER
+	bool
+	depends on !CPU_CORTEX_M
+	help
+	  This option indicates that the ARM CPU is connected to a custom (i.e.
+	  non-GIC) interrupt controller.
+
+	  A number of Cortex-A and Cortex-R cores (Cortex-A5, Cortex-R4/5, ...)
+	  allow interfacing to a custom external interrupt controller and this
```

Comment:
Is the help here still relevant for arm64?

Reply:
yes, it is.